pkg/router/functionHandler.go

```diff
@@ -501,21 +503,19 @@ func (roundTripper RetryingRoundTripper) addForwardedHostHeader(req *http.Reques
 }
 
 // unTapservice marks the serviceURL in executor's cache as inactive, so that it can be reused
-func (fh functionHandler) unTapService(fn *fv1.Function, serviceUrl *url.URL) error {
+func (fh functionHandler) unTapService(fn *fv1.Function, serviceURL *url.URL) {
```

Comment:
Was removing returning `error` intentional? CC @therahulbhati

Reply:
Yes. The returned error is ignored by the method's only consumer.

Reply:
@gauravgahlot I would actually say that let the method return error. Whether to consume the error or ignore is up to the caller. Can we please make this final change in both PRs and I will merge them!

Reply:
I have updated both the PRs and made linter happy by adding the `//nolint errcheck` to the method calls.